geometry_msgs/Point pose
time stamp
bool door_status
string id 